src/apps/mpx.erl

```diff
@@ -79,8 +83,8 @@ handle_call({subscribe, #de_client{meta=Apps, data=AppData}=Client, App}, _, Sta
               gen_server:cast(?SERVER, {reply_to_client, NewClient, element(3, Tuple)});
```

Comment:
Line 83 is too long:               gen_server:cast(?SERVER, {reply_to_client, NewClient, element(3, Tuple)});.